specification/metrics/amortizedcost.md

```diff
@@ -0,0 +1,42 @@
+# Amortized Cost
+
+Amortized cost represents the sum of the amortization of upfront fees, the amortization of recurring fees, and the discounted variable rate of the line item. This column is often used to show spending trends that include on demand rates, effective commitment-based discount rates, recurring commitments, as well as amortization of upfront fees. The fee and upfront amortization portion of this column’s value should be proportional to the number of units for the line item and the time granularity of the data. The currency for the value specified in Billed Cost can be found in the [Billing Currency](#billingcurrency) column.
```

Comment:
In the definition we list various elements that are included in the amortized cost. I'm not sure if that's implied or if we should also mention the basic, billed cost...